### PR TITLE
fix: split marker to ui_marker and mod_input_marker for test cases

### PR DESCRIPTION
**Pull request by kgwizdz-splunk**
UI tests may be slow to execute so we need to explore adding the ability to use markers that can speed up test execution by parallelizing UI tests. This is currently implemented only for modinput tests ([configuration example can be found here](https://github.com/splunk/splunk-add-on-for-amazon-web-services/blob/84f5ec10221dc2d660c824c7b908dce58eea9cfa/.github/workflows/build-test-release.yml#L32-L33)).

Example workflow execution: [https://github.com/splunk/splunk-add-on-for-amazon-web-services/pull/1220](https://github.com/splunk/splunk-add-on-for-amazon-web-services/pull/1220
)
Updated test runs: 
- https://github.com/splunk/splunk-add-on-for-amazon-web-services/actions/runs/9254951314
- https://github.com/splunk/test-addonfactory-repo/actions/runs/9254997916

### CODE REVIEW

**Comment by kgwizdz-splunk** [comment]
@mkolasinski-splunk Sure, added to description

**Comment by artemrys** [comment]
Just to make sure we are on the same page - if you want this change to be released - let's have this PR be against `main` branch and use `fix` instead of `feat`. So it can be seamlessly integrated into all the TAs.

**Comment by artemrys** [comment]
@kgwizdz-splunk please resolve conflicts.

Let's avoid making this breaking change - let's not rename the existing marker.

**Comment by artemrys** [comment]
@kgwizdz-splunk please check out `pre-commit` failure

File: .github/workflows/reusable-build-test-release.yml [hunk 1]
```diff
@@ -1493,7 +1499,12 @@ jobs:
         splunk: ${{ fromJson(needs.meta.outputs.matrix_combinedSplunkversion) }}
         browser: [ "chrome" ]
         vendor-version: ${{ fromJson(needs.meta.outputs.matrix_supportedUIVendors) }}
-
+        python39: [false]
+        marker: ${{ fromJson(inputs.ui_marker) }}
+        include:
+            - splunk: ${{ fromJson(needs.meta.outputs.python39_splunk) }}
+              browser: "chrome"
+              python39: true
```
Comment (artemrys) [inline .github/workflows/reusable-build-test-release.yml:1507]: Why this change is included in this PR?

Comment (kgwizdz-splunk) [reply .github/workflows/reusable-build-test-release.yml:1507]: strange apparently at rebase I inadvertently added something, it is enough if this line remains:
`marker: ${{ fromJson(inputs.ui_marker) }}` yes?

Comment (artemrys) [reply .github/workflows/reusable-build-test-release.yml:1507]: yes, that should be fine

File: .github/workflows/reusable-build-test-release.yml [hunk 2]
```diff
@@ -4,7 +4,13 @@ on:
     inputs:
       marker:
```
Comment (mkolasinski-splunk) [inline .github/workflows/reusable-build-test-release.yml:5]: let's make it modinput_marker

File: .github/workflows/reusable-build-test-release.yml [hunk 3]
```diff
@@ -4,7 +4,13 @@ on:
...
         required: false
-        description: 'Parallel run marker'
+        description: 'Parallel run mod_input marker'
```
Comment (mkolasinski-splunk) [inline .github/workflows/reusable-build-test-release.yml:7]:
```suggestion
        description: 'Parallel run modinput marker'
```

File: .github/workflows/reusable-build-test-release.yml [hunk 4]
```diff
@@ -17,7 +23,7 @@ on:
         required: false
         description: "branch for k8s manifests to run the tests on"
         type: string
-        default: "v1.0.0"
+        default: "v1.1.0"
```
Comment (mkolasinski-splunk) [inline .github/workflows/reusable-build-test-release.yml:26]: should this change be really part of this PR?

File: .github/workflows/reusable-build-test-release.yml [hunk 5]
```diff
@@ -1568,7 +1586,7 @@ jobs:
         with:
           splunk: ${{ matrix.splunk.version }}${{ secrets.OTHER_TA_REQUIRED_CONFIGS }}
           test-type: ${{ env.TEST_TYPE }}
-          test-args: ${{ env.TEST_ARGS }}
+          test-args: ${{ env.TEST_ARGS }} ${{ steps.create-test-arg.outputs.test-arg }} ${{ matrix.marker }}
           job-name: ${{ steps.create-job-name.outputs.job-name }}
```
Comment (mkolasinski-splunk) [inline .github/workflows/reusable-build-test-release.yml:1590]: Isn't test-browser input missing here?

**Review by mkolasinski-splunk — CHANGES_REQUESTED** [review]

File: .github/workflows/reusable-build-test-release.yml (hunk 5 above)
Comment (mkolasinski-splunk) [reply .github/workflows/reusable-build-test-release.yml:1590]: @kdoroszko-splunk Or rather you want to onboard this change with another PR?

File: .github/workflows/reusable-build-test-release.yml (hunk 4 above)
Comment (kdoroszko-splunk) [reply .github/workflows/reusable-build-test-release.yml:26]: Right now we have v2.0.0 on main branch. There will be conflict which will have to be resolved either way.

File: .github/workflows/reusable-build-test-release.yml (hunk 5 above)
Comment (kdoroszko-splunk) [reply .github/workflows/reusable-build-test-release.yml:1591]: Let's merge this PR. I've added the argument here: [line1](https://github.com/splunk/addonfactory-workflow-addon-release/pull/261/files#diff-bdb402178d2b76aa0f8acc15af0ee19e0bd76fd53028475dabe5503b5fe874dcR1514) [line2](https://github.com/splunk/addonfactory-workflow-addon-release/pull/261/files#diff-bdb402178d2b76aa0f8acc15af0ee19e0bd76fd53028475dabe5503b5fe874dcR1601). 
I will also have to change splunk/wfe-test-runner-action@v1.6 version to v5.0 after merging this [PR](https://github.com/splunk/wfe-test-runner-action/pull/27).

Comment (mkolasinski-splunk) [reply .github/workflows/reusable-build-test-release.yml:1591]: @kdoroszko-splunk please update documentation as well accordingly.

File: .github/workflows/reusable-build-test-release.yml (hunk 2 above)
Comment (kgwizdz-splunk) [reply .github/workflows/reusable-build-test-release.yml:5]: @artemrys  suggested "Let's avoid making this breaking change - let's not rename the existing marker."

File: .github/workflows/reusable-build-test-release.yml (hunk 5 above)
Comment (kdoroszko-splunk) [reply .github/workflows/reusable-build-test-release.yml:1591]: @mkolasinski-splunk, done. Take a look.

**Review by mkolasinski-splunk — APPROVED** [review]
